src/components/molecules/Hero/Hero.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import sth from './img.png';
+import {HeroWrapper, HeroContent} from './styles';
+
+const Hero = () => {
+    return (<HeroWrapper>
+        <HeroContent>
+            <img src={sth}
```

Comment:
@aryab2003 Code it don't just put the image.

Reply:
ok..can you help  as to how would i go about doing that?

Reply:
Write the text as headings and use button components.

Reply:
ok